alchemy/src/cloudflare/wrangler.json.ts

```diff
@@ -513,6 +513,8 @@ function processBindings(
       });
     } else if (binding.type === "ai_gateway") {
       // no-op
+    } else if (binding.type === "version_metadata") {
+      // no-op
```

Comment:
why is this a no-op, it should be included in the wranglerjson spec, update the type and include it